torch/onnx/errors.py

```diff
@@ -0,0 +1,44 @@
+"""ONNX exporter exceptions."""
+
+from typing import Optional
+
+from torch.onnx import _constants
+
+__all__ = ["OnnxExporterError", "CheckerError", "UnsupportedOperatorError"]
```

Comment:
Is it necessary to define `__all__` for this file?

Reply:
Since we have `from typing import Optional`, the CI check will complain w/o all, saying `Optional` doesn't belong to the module